packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNodeUtils.ts

```diff
@@ -184,23 +173,3 @@ export function parseSummaryForSubtrees(baseSummary: ISnapshotTree): ISubtreeInf
 		childrenPathPart: undefined,
 	};
 }
-
-/**
- * Checks if the summary contains .channels subtree where the children subtrees
- * would be located if exists.
- * @param baseSummary - summary to check
- */
-export function parseSummaryTreeForSubtrees(summary: ISummaryTree): ISubtreeInfo<SummaryObject> {
```

Comment:
while this may have only been used in one place in `summarizerNode.ts`, this does look like a general utility function that may be useful in future for other scenarios! Isn't it?

Reply:
Currently, when we write summaries, the data store summary tree is written under a ".channels" sub-tree. Same goes for DDSes.
When we were tranistioning to that pattern of summary trees, this function was used to detect whether we are writing data store / DDS summary under ".channels" or not.
It's not needed anymore since we always write under ".channels".
We still have to be able to detect this when loading from a snapshot and the function above this `parseSummaryForSubtrees` does this. Note the difference between the two functions - the one above uses `ISnapshotTree` (that is downloaded or read) and this one uses `ISummaryTree` (that is uploaded or written).